test/expectations/expectations.go

```diff
@@ -123,3 +126,22 @@ func ExpectDeleted(ctx context.Context, c client.Client, objects ...client.Objec
 		Expect(c.Get(ctx, client.ObjectKeyFromObject(o), o)).To(Or(Succeed(), MatchError(ContainSubstring("not found"))))
 	}
 }
+
+func ExpectCleanedUp(ctx context.Context, c client.Client, objects ...client.Object) {
```

Comment:
Do we want to add an ExpectNotFound at the end to make sure it's all cleaned up before the function ends?

Reply:
I can't call ExpectNotFound, because I'm acting on an object list -- there's no way to assert "expect none exist" without passing an object list, which I was struggling with.